sqldelight-core/src/main/kotlin/com/squareup/sqldelight/core/compiler/model/BindableQuery.kt

```diff
@@ -24,14 +24,18 @@ import com.squareup.sqldelight.core.lang.IntermediateType.SqliteType.ARGUMENT
 import com.squareup.sqldelight.core.lang.IntermediateType.SqliteType.NULL
 import com.squareup.sqldelight.core.lang.psi.InsertStmtMixin
 import com.squareup.sqldelight.core.lang.util.argumentType
+import com.squareup.sqldelight.core.lang.util.childOfType
 import com.squareup.sqldelight.core.lang.util.columns
 import com.squareup.sqldelight.core.lang.util.findChildrenOfType
 import com.squareup.sqldelight.core.lang.util.interfaceType
 import com.squareup.sqldelight.core.lang.util.table
 
 open class BindableQuery(
+  internal val identifier: PsiElement?,
```

Comment:
Should be possible to make this typesafe `SqlDelightStmtIdentifier`

Reply:
At some point I have to pass a `SqliteViewName` into `NamedQuery` constructor, and that doesn't seem to extend `SqlDelightStmtIdentifier`